.travis.yml

```diff
@@ -4,7 +4,7 @@ rust:
   - nightly
 
 install:
-  - travis_retry curl -Lf https://github.com/rust-lang-nursery/mdBook/releases/download/v0.1.7/mdbook-v0.1.7-x86_64-unknown-linux-gnu.tar.gz | tar -xz --directory=$HOME/.cargo/bin
+  - travis_retry curl -Lf https://github.com/rust-lang-nursery/mdBook/releases/download/v0.2.1/mdbook-v0.2.1-x86_64-unknown-linux-gnu.tar.gz | tar -xz --directory=$HOME/.cargo/bin
```

Comment:
Oh, hmm, this needs to be 0.2.3.  @steveklabnik would it be possible to publish a binary for the latest release on github?